docs/changelog.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+### 18-November-2020 - 12:58 CEST
+
+- [fix] Notify unexpected errors to slack channel (add link to message).
+- [job] AutomaticMerge: Fix PRs blocked by non team member users.
+- [bug] Build everything but OK or INVALID_CONFIG.
+- [fix] Do not use `--all` argument with `conan upload` when the package ID is given.
+- [fix] Fix error getting properties when the recipe doesn't have options.
+- [job] Tapaholes: Propose new profile set including new compiler configurations.
```

Comment:
Does this mean that Tapaholes is ready for prime time? 😍 
Or is this another iterative step towards it? :)

Reply:
We have run it in our staging environment where we have some recipes but of course not the amount we have in production. We hope to get it running soon although there would be some details to polish while we try it. But we could say we are close 😄

Reply:
And the thing that scares me most: what if recipes start to fail with the new compilers 🙈 ? We cannot enable the new profiles for regular jobs (pull-requests) until we have binaries available for the requirements... and we'd need help from the community if we need to fix hundreds of recipes. 

But this is not going to happen, everything will be smooth. Piece of cake.

Reply:
I don't think there is a clear solution for this

It should be transparent in some way, what is bad is a situation like with the new shared+MT runtime builds: Someone opens a pull request changing something and suddenly gets errors. That is confusing since you first search for the problem in your own changes. If it is clear "Hey, this configuration was never tested before!" you know where to start looking

Reply:
as I can see, after running tapa-holes for the first time, it will fail for a number of recipes, which were never tested for new compilers or compiler versions.
this is more or less expected - new versions of compilers are usually more strict, also older versions of libraries were never tested against newly released compilers. 
so what can we do?
I think the easiest solution is to add the following snippet for each failed recipe and version:
```
def configure(self):
    if self.version == <failed_version> and self.settings.compiler == <new compiler> and self.settings.compiler.version == <new compiler version>:
        raise ConanInvalidConfiguration("FIXME : library FooBar v%s doesn't build for new configuration %s %s" % (self.version, self.settings.compiler, self.settings.compiler.version)
```
after we update failed recipes, we need to re-run tapa-holes one more time (e.g. "fixing" `zlib` will unblock `boost`, and fixing `boost` will unblock `folly`, and so on).

then, contributors will have opportunity to remove added snippet and provide actual fixes (e.g. patches or whatever else is needed to fix recipe).

technically, it may require several (3-10) cycles, depending on how many recipes will fail and how deep are their graphs.